### PR TITLE
Revert change that broke hostname

### PR DESCRIPTION
**Pull request by areddish**
Revert c1cee163b94aaffbf6ab5d8cac05f5f571c6e8fd for Computer and Custom Vision.

That change defaulted the host to an invalid url which has all clients return 404 unless they override the base url.

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/ComputerVision.json [hunk 1]
```diff
@@ -1545,6 +1553,33 @@
...
+      "name": "AzureRegion",
+      "description": "Supported Azure regions for Cognitive Services endpoints",
+      "x-ms-parameter-location": "client",
+      "required": true,
+      "type": "string",
+      "in": "path",
+      "x-ms-skip-url-encoding": true,
+      "x-ms-enum": {
+        "name": "AzureRegions",
+        "modelAsString": false
```
Comment (dsgouda) [inline specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/ComputerVision.json]: Please acknowledge that marking `modelAsString` as false may result in breaking changes if new values are introduced in the enum set.

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please update the corresponding examples
(especially the modified URLs)

File: specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/ComputerVision.json (hunk 1 above)
Comment (areddish) [reply specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/ComputerVision.json]: Acknowledged, I'm just reverting things back to how they were :).

**Comment by areddish** [comment]
There are no examples to update, the offending commit did not update the examples to the new uri so the old uri persists.

**Comment by dsgouda** [comment]
Thanks for the feedback

**Review by dsgouda — APPROVED** [review]

**Comment by dsgouda** [comment]
LGTM

**Pull request merged**